cwms-data-api/src/main/java/cwms/cda/api/Controllers.java

```diff
@@ -132,8 +130,7 @@ public final class Controllers {
     static final String SPECIFIED_LEVEL_ID = "specified-level-id";
 
     static {
-        JavalinValidation.register(JooqDao.DeleteMethod.class,
-            Controllers::getDeleteMethod);
+        JavalinValidation.register(JooqDao.DeleteMethod.class, Controllers::getDeleteMethod);
```

Comment:
I take it just doing this once in ApiServlet did not do the trick?

Reply:
It should but the tests don't go through ApiServlet.  The registration probably should be its own method, then the test could call it.  Will move the registration test into ApiServletTest which makes more sense.

Reply:
Okay, something like that you can do in a BeforeTest handler but I don't think this will harm anything so we can experiment with it later.